src/js/edu-benefits/definitions/serviceBefore1977.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+
+export const schema = commonDefinitions.serviceBefore1977;
+
+export const uiSchema = {
+  married: {
+    'ui:title': 'Are you currently married?',
+    'ui:widget': 'yesNo'
+  },
+  haveDependents: {
+    'ui:title': 'Do you have any children who fall into these categories?',
+    'ui:description': () => (
```

Comment:
If it's just a string of the markup, React will escape it and you'll see text with the markup in it (e.g. `<div>description text</div>`. I just pushed a change to allow regular JSX elements, which seems more consistent with what works in other parts of the config. Before I was allowing either a string or a React component (which is a function).